contracts/Pool.sol

```diff
@@ -26,53 +26,53 @@ contract Pool is PoolFDT {
+    uint256 private immutable liquidityAssetDecimals;  // The precision for the Liquidity Asset (i.e. `decimals()`).
 
-    uint256 public           stakingFee;   // Fee for stakers   (in basis points)
-    uint256 public immutable delegateFee;  // Fee for delegates (in basis points)
+    uint256 public           stakingFee;   // The fee Stakers earn            (in basis points).
+    uint256 public immutable delegateFee;  // The fee the Pool Delegate earns (in basis points).
 
-    uint256 public principalOut;  // Sum of all outstanding principal on Loans
-    uint256 public liquidityCap;  // Amount of liquidity tokens accepted by the Pool
-    uint256 public lockupPeriod;  // Period of time from an account's depositDate that they cannot withdraw any funds
+    uint256 public principalOut;  // The sum of all outstanding principal on Loans.
+    uint256 public liquidityCap;  // The amount of liquidity tokens accepted by the Pool.
+    uint256 public lockupPeriod;  // The period of time from an account's deposit date during which they cannot withdraw any funds.
 
     bool public openToPublic;  // Boolean opening Pool to public for LP deposits
 
     enum State { Initialized, Finalized, Deactivated }
     State public poolState;
 
     mapping(address => uint256)                     public depositDate;                // Used for withdraw penalty calculation.
-    mapping(address => mapping(address => address)) public debtLockers;                // Address of the `DebtLocker` contract corresponding to [Loan][DebtLockerFactory].
-    mapping(address => bool)                        public poolAdmins;                 // Pool Admin addresses that have permission to do certain operations in case of disaster management..
+    mapping(address => mapping(address => address)) public debtLockers;                // Address of the DebtLocker corresponding to `[Loan][DebtLockerFactory]`.
+    mapping(address => bool)                        public poolAdmins;                 // The Pool Admin addresses that have permission to do certain operations in case of disaster management.
     mapping(address => bool)                        public allowedLiquidityProviders;  // Mapping that contains the list of addresses that have early access to the pool.
-    mapping(address => uint256)                     public withdrawCooldown;           // Timestamp of when LP calls `intendToWithdraw()`.
-    mapping(address => mapping(address => uint256)) public custodyAllowance;           // Amount of PoolFDTs that are "locked" at a certain address.
-    mapping(address => uint256)                     public totalCustodyAllowance;      // Total amount of PoolFDTs that are "locked" for a given account, cannot be greater than balance.
+    mapping(address => uint256)                     public withdrawCooldown;           // The timestamp of when individual LP have notified of they intent to withdraw.
```

Comment:
`// The timestamp of when individual LPs have notified of their intent to withdraw.`